monkey/monkey_island/cc/resources/monkey.py

```diff
@@ -1,19 +1,32 @@
 import json
-from datetime import datetime
+from datetime import datetime, timedelta
 
 import dateutil.parser
-from flask import request
 import flask_restful
+from flask import request
 
+from monkey_island.cc.models.monkey_ttl import MonkeyTtl
 from monkey_island.cc.database import mongo
 from monkey_island.cc.services.config import ConfigService
 from monkey_island.cc.services.node import NodeService
 
+MONKEY_TTL_EXPIRY_DURATION_IN_SECONDS = 120
+
 __author__ = 'Barak'
 
 # TODO: separate logic from interface
 
 
+def create_monkey_ttl():
```

Comment:
why not move this function to MonkeyTtl?

Reply:
It's not really part of the model, it's the usage... Seems to me like it belongs here more, at least until we do a general overhaul in which we might create an intermediate level of saving and retrieving data related to its actual usage logic and not its structure

If you disagree I'll move it :+1: